workers/nomad-job-specs/create_quantpendia.nomad.tpl

```diff
@@ -0,0 +1,99 @@
+job "CREATE_QUANTPENDIA" {
+  datacenters = ["dc1"]
+
+  type = "batch"
+
+  # These are high priority because if we've dispatched them,
+  # we need them.
+  priority = 80
+
+  parameterized {
+    payload       = "forbidden"
+    meta_required = ["JOB_NAME", "JOB_ID"]
+  }
+
+  group "jobs" {
+    restart {
+      attempts = 0
+      mode = "fail"
+    }
+
+    reschedule {
+      attempts = 0
+      unlimited = false
+    }
+
+    ephemeral_disk {
+      size = "10"
+    }
+
+    task "create_quantpendia" {
+      driver = "docker"
+
+      kill_timeout = "30s"
+
+      # This env will be passed into the container for the job.
+      env {
+        ${{AWS_CREDS}}
+        DJANGO_SECRET_KEY = "${{DJANGO_SECRET_KEY}}"
+        DJANGO_DEBUG = "${{DJANGO_DEBUG}}"
+
+        DATABASE_NAME = "${{DATABASE_NAME}}"
+        DATABASE_USER = "${{DATABASE_USER}}"
+        DATABASE_PASSWORD = "${{DATABASE_PASSWORD}}"
+        DATABASE_HOST = "${{DATABASE_HOST}}"
+        DATABASE_PORT = "${{DATABASE_PORT}}"
+        DATABASE_TIMEOUT = "${{DATABASE_TIMEOUT}}"
+
+        RAVEN_DSN="${{RAVEN_DSN}}"
+        RAVEN_DSN_API="${{RAVEN_DSN_API}}"
+
+        NOMAD_HOST = "${{NOMAD_HOST}}"
+        NOMAD_PORT = "${{NOMAD_PORT}}"
+
+        RUNNING_IN_CLOUD = "${{RUNNING_IN_CLOUD}}"
+
+        USE_S3 = "${{USE_S3}}"
+        S3_RESULTS_BUCKET_NAME = "${{S3_RESULTS_BUCKET_NAME}}"
+        S3_BUCKET_NAME = "${{S3_BUCKET_NAME}}"
+        LOCAL_ROOT_DIR = "${{LOCAL_ROOT_DIR}}"
+        MAX_DOWNLOADER_JOBS_PER_NODE = "${{MAX_DOWNLOADER_JOBS_PER_NODE}}"
+
+        ELASTICSEARCH_HOST = "${{ELASTICSEARCH_HOST}}"
+        ELASTICSEARCH_PORT = "${{ELASTICSEARCH_PORT}}"
+
+        LOG_LEVEL = "${{LOG_LEVEL}}"
+      }
+
+      # The resources the job will require.
+      resources {
+        # CPU is in AWS's CPU units.
+        cpu =   4000
+        # Memory is in MB of RAM. Instance has 1,952GB of RAM.
+        memory = 8192
```

Comment:
This is going to be enough for all organisms? I mean... I can see it cause we don't have to store too much data in memory at any point. It just seems really low, but it might be appropriate.

Reply:
Not sure, seemed enough since the job will only download quant files and the compression should not take more than 2gb of ram. Do you think we should increase it?

Reply:
Not necessarily. Do you know how large the metadata object for human will be? I think that might be the peak RAM usage actually.

The other thing to consider is that number of quantpendia jobs running on a single machine will be `(1000 or 2000 depending on instance type) / 8)` and the machine will only have 9TB of disk space available to write the files to. Would 125 or 250 of these jobs running simultaneously cause the volume to fill up?

Reply:
In terms of disk space, we had calculated that 5TB should be enough for all organism quantpendias. (ref https://github.com/AlexsLemonade/refinebio/pull/1642)

The metadata was about 3.3Gb for the human, which is probably the biggest one.

```
$ du -sh /var/ebs/smashed/714ec9dc-b5c4-4465-8a45-d46c173d3252/output/*
3.3G    /var/ebs/smashed/714ec9dc-b5c4-4465-8a45-d46c173d3252/output/aggregated_metadata.json
958G    /var/ebs/smashed/714ec9dc-b5c4-4465-8a45-d46c173d3252/output/HOMO_SAPIENS
20K    /var/ebs/smashed/714ec9dc-b5c4-4465-8a45-d46c173d3252/output/LICENSE.TXT
4.0K    /var/ebs/smashed/714ec9dc-b5c4-4465-8a45-d46c173d3252/output/README.md
```

I think 8GB should be enough, but we can change it. What size do you think would be more appropiate?

Reply:
Nah all that sounds good, I just wanted to make sure you had thought it through.